src/elements/content-picker/shareAccessCellRenderer.js

```diff
@@ -15,11 +16,16 @@ export default (
     extensionsWhitelist: string[],
     hasHitSelectionLimit: boolean,
 ) => ({ rowData }: { rowData: BoxItem }) => {
-    if (!isRowSelectable(selectableType, extensionsWhitelist, hasHitSelectionLimit, rowData)) {
+    if (!isRowSelectable(selectableType, extensionsWhitelist, hasHitSelectionLimit, rowData) || !rowData.selected) {
         return <span />;
     }
 
-    return (
+    const { allowed_shared_link_access_levels } = rowData;
+    const isLoading = !allowed_shared_link_access_levels;
```

Comment:
Minor, but do we need a failed state here now that it's a separate call?

Reply:
@ConradJChan, thoughts on @jeremypress's comment above?

Reply:
Yeah I checked with Zach and Rohit and since it seems like an uncommon error, just leaving it as is for now